proxy/proxy.go

```diff
@@ -70,8 +71,15 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	defer logRequest()
 
+	// PRI requests are an internal check of HTTP/2 which we just have to
+	// say OK to.
+	if r.Method == "PRI" {
```

Comment:
Some more info on this would be nice. What problem do we have if we don't answer?

Reply:
Removed the whole commit. We did try out some stuff with this but I think in the end (with TLS turned off, this is not needed anymore). Same with the following 2 commits.